src/AddTodoForm.jsx

```diff
@@ -1,17 +1,41 @@
-function AddTodoForm(todoprops) {
+import {useState} from 'react';
+
+function AddTodoForm({ onAddTodo, todoList }) {
+    // New state variable and function to handle title change
+    const [todoTitle, setTodoTitle] = useState("");
+
+    function handleTitleChange(event) {
+        console.log(todoTitle);
+        event.preventDefault();
```

Comment:
you do not need to use preventDefault here because this function  does not related with reloading behavior ([documentation](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault))

Reply:
preventDefault is removed, thank you